src/lib.rs

```diff
@@ -729,6 +735,9 @@ impl Format {
             Format::Repeat(_) => Bounds::new(0, None),
             Format::Repeat1(f) => f.lookahead_bounds(module) * Bounds::new(1, None),
             Format::RepeatCount(expr, f) => f.lookahead_bounds(module) * expr.bounds(),
+            Format::RepeatBetween(xmin, xmax, f) => {
+                f.lookahead_bounds(module) * Bounds::union(xmin.bounds(), xmax.bounds())
+            }
```

Comment:
`lookahead_bounds` might be a bit weird for `RepeatCount` and `RepeatBetween` maybe? I need to think about that one 🤔

Reply:
Right...let me know if you happen to figure out what it should be (assuming that doesn't happen to be what it is at the moment)...

Reply:
I guess it's only a conservative over-estimate, so it's fine. But if you had `RepeatCount(expr, f)` then while `match_bounds` is `f.match_bounds(module) * expr.bounds()`, `lookahead_bounds` could be tighter than `f.lookahead_bounds(module) * expr.bounds()`, it could be something like `f.match_bounds() * (expr - 1) + f.lookahead_bounds()`, where we add up all the bytes we actually read to match the first N-1 repetitions and then the bytes we looked ahead for the last one (which will be >= the number of bytes we matched). However this is a completely unnecessary thing to do, so let's not do it.